### PR TITLE
[#155646523] Scripts and configs supporting new ECS deployment model

### PR DESCRIPTION
**Pull request by brainsik**
* Updates the DB migrations script
* Replaces `ecs-deploy` with new script to deploy the app container
* Updates CircleCI config to use new scripts

### CODE REVIEW

**Comment by brainsik** [comment]
## Important
This depends on rolling out the new infrastructure. Currently, this runs agains an experimental environment (called "experimental"). Once the new staging environment is rolled out, this can be updated to run against staging and merged in.

File: .circleci/config.yml [hunk 1]
```diff
@@ -126,10 +126,10 @@ jobs:
       - checkout
       - run:
           name: Snapshot database
-          command: bin/rds-snapshot-app-db staging
+          command: bin/rds-snapshot-app-db experimental
```
Comment (dynamike) [inline .circleci/config.yml]: Are you planning to switch this back to the staging site or leave it in experimental?

File: .circleci/config.yml [hunk 2]
```diff
@@ -210,13 +208,13 @@ workflows:
             - build_migrations
           filters:
             branches:
-              only: master
+              only: bsik-ecs-refactor
```
Comment (dynamike) [inline .circleci/config.yml]: Will this also get reverted back to `master` at some point?

File: bin/ecs-deploy-app-container [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+readonly green_task_def_arn
+check_arn "$green_task_def_arn"
+
+if update_service "$green_task_def_arn"; then
+    echo "Success."
+    exit 0
+fi
+echo "Service failed to stabilize!"
+
+echo "* Rolling back to $blue_task_def_arn"
```
Comment (dynamike) [inline bin/ecs-deploy-app-container]: Have you tested a rollback?

**Comment by dynamike** [comment]
I missed your comment about the experimental testing. So ignore those comments

File: bin/ecs-deploy-app-container (hunk 3 above)
Comment (brainsik) [reply bin/ecs-deploy-app-container]: Hella tested. :-)

File: .circleci/config.yml (hunk 2 above)
Comment (macrael) [inline .circleci/config.yml]: Is this meant to be an experiment? We should coordinate merging this so that our demo work is not interrupted.

Comment (brainsik) [reply .circleci/config.yml]: I have a big comment marked "Important" above: https://github.com/transcom/mymove/pull/217#issuecomment-370985644. You're the second person who's commented on this, so that clearly didn't work. Next time I'll directly comment the code.

tl;dr - this is running experimental infrastructure. I'll make a commit that puts it back on master when the new staging environment is brought up. Aiming to do that today.
